js/dactylo.js

```diff
@@ -110,8 +104,8 @@ window.addEventListener('DOMContentLoaded', () => {
   };
 
   gLayout.addEventListener('change', () => {
-    localStorage.setItem('layout', gLayout.value);
-    fetchLayout().then(setLessonLevel);
+    window.location.hash = `${gLayout.value}`;
+    init();
```

Comment:
`init` is not a good function name, especially in this context. 

Besides, I’m not sure I see the point in storing the layout in both the URL hash and the localStorage.

Reply:
I thought that we needed the layout in local storage, to remember it if someone bookmarks the page without the hash in the url, but I’m not sure it’s a good idea. We may only need the url hash